### ISSUE TITLE
[Feature]: NISAR processing

### ISSUE THREAD

**Issue by pederao** [gmtsar/gmtsar#907]
**NISAR processing pipeline**
I think I heard in a talk that there are plans to provide a processing pipeline for data from the NISAR satellite in the same vein that a processing pipeline for Sentinel-1 data exists already.  Is there a time-line for this and are there some documents ready for how to do this already?  In either case I would be excited to see such a pipeline.

**Comment by Xiaohua-Eric-Xu** [comment]
Glad you are asking. I wrote a pre-processor for NISAR using UAVSAR simulated NISAR dataset, and we are testing it with the more recent Sentinel-1 and ALOS-2 simulated NISAR data. The code is currently at GMTSAR/preproc/NSR_preproc. However, due to some reason, the launch of NISAR is postponed again maybe to Oct this year. So unfortunately it's very unlikely that we'll get any real data before 2025. 

Once the data is ready, we are planning to build the pipeline to the current p2p_processing and batch_processing scripts.
